rst_files/lln_clt.rst

```diff
@@ -214,26 +214,26 @@ Setup
 
 .. code-block:: julia
 
-    using LinearAlgebra, Statistics, Compat 
+    using LinearAlgebra, Statistics, Compat
     using StatPlots, Distributions, Random, Statistics
     gr(fmt = :png, size = (900, 500))
 
 .. code-block:: julia
 
     function ksl(distribution, n = 100)
         title = nameof(typeof(distribution))
-        observations = rand(distribution, n)
-        sample_means = cumsum(observations) ./ (1:n)
+        X_i = rand(distribution, n)
+        X̄_n = cumsum(X_i) ./ (1:n)
```

Comment:
We need to be careful with this --- I think we ruled against unicode glyphs (combinations of symbols, like letters and bars/hats/underscores) in the style guide.